tests/test_cli/test_cfg_subcmd.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import os
+import shutil
+
+from test_cli import capture
+
+def test_cfg_help(python_version):
+    """Test if Manim successfully adds configparsers when a subcommand is invoked."""
+    os.chdir(os.path.dirname(__file__))
```

Comment:
**Issue** : as we have discussed on Discord, I don't think doing this is a good idea. 
It works right now as you set back the right `test_directory` in `test_cli.py` so tests will work fine as `test_cli `is ran directly after `test_cfg_subcmd` by pytest, but won't when we will add other tests that will run between `test_cli `and `test_cfg_subcmd`.
Why could you not execute a command like that in the test to move to the right directory : 
`cd <os.path.dirname(__file__)> && python -m manim cdg --help` ?
Shouldn't this work?

Reply:
> It works right now as you set back the right test_directory in test_cli.py so tests will work fine as test_cli is ran directly after test_cfg_subcmd by pytest, but won't when we will add other tests that will run between test_cli and test_cfg_subcmd.

> Why could you not execute a command like that in the test to move to the right directory :
`cd <os.path.dirname(__file__)> && python -m manim cfg --help` ?

Well, yes, that would definitely work. My only gripe with that is that the same command will have to be added for all of the `cfg` subcommand tests separately, but that is not much of an issue.